phpstan.neon.dist

```diff
@@ -1,6 +1,8 @@
 parameters:
 	level: 5
 	inferPrivatePropertyTypeFromConstructor: true
+	bootstrapFiles:
+	    - ./vendor/symfony/dependency-injection/Loader/Configurator/functions.php
```

Comment:
something is odd with the phpstan setup. This is normally not needed. Maybe its because its an outdated version (v2 is out since a while) or because its separate in the tools directory. Not sure.

Without this we get
<img width="677" height="425" alt="image" src="https://github.com/user-attachments/assets/f3133590-d5b0-46ab-ac45-e0e821f1ec1a" />